src/steps/ms-defender/machine/converters.ts

```diff
@@ -168,10 +205,7 @@ const formatValidMacAddress = (macAddress: string) => {
     /([0-9A-Fa-f]{2})(?=[0-9A-Fa-f]{2})/g,
     '$1:',
   );
-  return [
-    formattedMacAddress.toLowerCase(), // Lowercase to match the macAddress of aws_ami(s)..
-    formattedMacAddress.toUpperCase(), // Uppercase because that is the industry standard thus making J1QL easier.
-  ];
+  return formattedMacAddress.toUpperCase(); // Uppercase because that is the industry standard thus making J1QL easier.
```

Comment:
Because we have the normalizer now, we no longer need to provide MACs in both uppercase and lowercase.